crates/unparser/test_files/function_def.py

```
@@ -0,0 +1,51 @@
+def name_4[*name_2](  # type: ignore
```

Pylint (3.3.7)

[error] Parsing failed: 'expected '(' (<unknown>, line 1)'

(E0001)